### ISSUE TITLE
Suggestion: Mediawiki

### ISSUE THREAD

**Issue by borgnetzwerk** [3b1b/captions#9]
Hello there,

TL;DR:
I'd suggest to transcribe via OpenAI Whisper, then translate in your prefered way, then push all transcripts on a mediawiki instance. To my knowledge, this is the best middle-ground for getting people involved, making it easy to revert / detect / manage vandalism, while simultaniously having the tools to easily divergy workload, rights, etc.
Then, after given thresholds (time passed, edits, overseer approval, ...), pull the transcripts from the wiki, convert to captions (and Text-to-speech - potentially, this review could be posted, discussed etc. again on the same wiki) - and progress to the video.

Long story:

Hi, my name is Tim, I am a PhD Student in Digital Libraries and Datascience and Co-founded the BorgNetzWerk in 2023 as a charity NPO to make accessible knowledge more connected. Some quick references:
* Our first in-use project: Transcripts for the history of philosophy, "without any gaps." podcast from Peter Adamson, Professor of Philosophy [at the LMU in Munich](https://www.philosophie.uni-muenchen.de/lehreinheiten/philosophie_6/personen/adamson/index.html) and at [King's College London](https://www.kcl.ac.uk/people/peter-adamson): https://historyofphilosophy.net/transcripts
* A pitch for a potential Wiki for CinemaTherapy:  https://www.reddit.com/r/cinema_therapy/comments/131zmgq/searchable_wiki_for_cinema_therapy/
    * And one example how a Transcript would look like: https://data.bnwiki.de/index.php?title=Cinema_Therapy:Therapist_Reacts_to_Everything_Everywhere_All_at_Once

We're almost done with the bureaucracy related to properly founding a charitable NPO in germany, with all major steps being done. Currently, we are reworking the online representation to properly represent the charity, but some artefacts are born english and/or somewhat tranlsated:
* Our code used for transcription, analysis and publishing of media (videos, text, etc.): https://github.com/borgnetzwerk/tools
* Our aims and values: https://borgnetzwerk.de/aims-and-values/
* Our founding as a charity: https://borgnetzwerk.de/founding/

I personally would love to help, from low level support like being one of the language support ppl, to co-designing, hosting and maintaining the wiki, community-effort, etc. Exactly this is also my PhD topic alongisde the reason for founding the NPO, so I'd love to help, first and foremost in the way most helpful to your requirements.

Long story short:
I suggest Mediawiki - and offer a bunch of additional ideas.

I hope this helps :)

Best
Tim
